cmd/generatetf/generator/generator_test.go

```diff
@@ -72,13 +72,11 @@ func TestGeneratorTerraform(t *testing.T) {
 				"categoryToContent": [{ "from": "from", "to": "to" }],
 				"legacyConversionPixelId": { "from": "from", "to": "to" },
 				"useNativeSDK": { "web": true },
-				"oneTrustCookieCategories": {
-				  "web": [
+				"oneTrustCookieCategories": [
```

Comment:
Why are we excluding the "web" source type from this? This behaviour shouldn't have changed, at least for the Facebook pixel destination that is being used in this test? Was the test failing anyway because it was not updated previously?

Reply:
Yes. Up until now, this field's data has not been source-type specific.

I'll fix this test once I make the relevant changes now in all the destinations.